README.md

```diff
@@ -53,6 +54,12 @@ Sorted by [year](#year) or [topics](#topics)
 
 ### Year
 
+#### 2019
+
+   - [[Butz2019](#butz2019)] [**Deep Convolutional Sum-Product Networks**](https://www.aaai.org/Papers/AAAI/2019/AAAI-ButzC.3622.pdf) *AAAI 2019* [`modeling`](#modeling)
+   - [[Molina2019](#molina2019)] [**SPFlow: An Easy and Extensible Library for Deep Probabilistic Learning using Sum-Product Networks**](https://arxiv.org/abs/1901.03704) *preprint* [`applications`](#applications)
```

Comment:
I listed Molina et al. SPFlow under `applications`, though it might fit under a couple topics.

Reply:
Better to link it to code as well